src/Webhooks/Events/OnLogin.cs

```diff
@@ -3,13 +3,14 @@
 
 namespace Webhooks.RegisteredEvents
 {
-    public class OnLogin : IScript
+    public class OnLogin : PlayerEvents
     {
-        [Target(GameSourceEvent.PlayerInitialize, ExecutionMode.Event)]
-        public void OnEvent(ShPlayer player)
+        [Execution(ExecutionMode.Event)]
+        public override bool Initialize(ShEntity entity)
         {
-            if(!player.isHuman) return;
-            Core.Instance.SendDefaultEvent(DefaultEvents.OnLogin, player.username);
+            if(!entity.Player.isHuman) return true;
```

Comment:
Can `entity.Player` be null?

Reply:
It can't since it's from PlayerEvents i think the entity must have Player